src/modules/browser-event-manager/browser-event-manager.ts

```diff
@@ -69,9 +70,6 @@ export class BrowserEventManager {
         this.onPointerMove(this.pointerMoveEvent);
       }
     });
-
-    // @todo temp.
-    this.activateEvents();
```

Comment:
this was not being unbound

Reply:
```
// @todo temp`
```
🤦 sorry.

Reply:
I just ran this in CanvasPanel, and it seems like no interactions are working :( ... so maybe this was doing something _(or the activation checking below)_, going to unwind it to see if this is the case

Reply:
So, turns out the issue is that in CanvasPanel, we need this... but in Atlas we don't